### ISSUE TITLE
Self-signed certificate on Collabora does not result in proper error message

### ISSUE THREAD

**Issue by matthijskooijman** [nextcloud/richdocuments#264]
I suspect this issue belongs in this repo. If not, I'll happily move it elsewhere.

TLDR; if you run Collabora with a self-signed certificate, you get a internal server error and a (seemingly unrelated) PHP fatal error. Refusing to run seems acceptable, but it would be good to show a better error message.

I realize that mixing HTTP and HTTPS and using self-signed certificates  is not recommended for production, but I was just intending to test. It is totally acceptable for this setup to not work, but it looks like there is a problem in the error handling, that might be worth fixing, so anyone else trying something like this will not be as confused as I was :-)

### Steps to reproduce
1. Install Next cloud & Collabora using their docker images
2. Modify the Collabora container to expose its port directly, rather than setting up a reverse proxy
3. Set up the richdocuments app in Nextcloud, pointing to the Collabora port directly (e.g. https://1.2.3.4:9980/)
4. Create and open an odt file.

### Expected behaviour
I would have hoped this setup to work. Failing that, I would have liked to see a proper error message directly.

### Actual behaviour
Opening up a odt file from nextcloud gives a mostly empty page. The url inside the iframe for the richtdocuments app (http://1.2.3.4:8080/apps/richdocuments/index?fileId=154&requesttoken=removed)  gives a 500 error. Looking in the docker logs, I see a fatal error:

```
[Tue Sep 18 10:05:04.189698 2018] [php7:error] [pid 416] [client 83.80.204.85:46282] PHP Fatal error:  Cannot declare class GuzzleHttp\\Handler\\CurlFactory, because the name is already in use in /var/www/html/3rdparty/guzzlehttp/guzzle/src/Handler/CurlFactory.php on line 16
83.80.204.85 - - [18/Sep/2018:10:05:04 +0000] "GET /apps/richdocuments/index?fileId=154&requesttoken=removed HTTP/1.1" 500 870 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0"
```

Later, I noticed another error in the logging section of the Nextcloud section, preceding each fatal error: GuzzleHttp\Exception\RequestException: cURL error 60: SSL certificate problem: self signed certificate in certificate chain (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)

My hypothesis is that this RequestException is caught, but not handled adequately, resulting in the subseqeunt PHP fatal error. Ideally, this fatal error would not be triggered, but either a proper error message is shown to the user, or the exception is triggered as a fatal error instead.

### Server configuration
**Operating system**: Debian host for Docker

**Web server:** Apache

**Database:** Sqlite

**PHP version:** 7.2.10

**Nextcloud version:** 14.0.0

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install

**Where did you install Nextcloud from:** Docker

**Signing status:** No errors have been found.

**List of activated apps:** richdocuments only

**Comment by matthijskooijman** [comment]
I looked a bit further, and found that this `RequestException` passes through this bit of (rather pointless?) code:

https://github.com/nextcloud/richdocuments/blob/master/lib/WOPI/DiscoveryManager.php#L83-L87

**Comment by matthijskooijman** [comment]
The original `RequestException` is caught here: https://github.com/nextcloud/richdocuments/blob/9decb9f4edac15a558295306d5887a8eec8672a2/lib/Controller/DocumentController.php#L128-L142

There it translates to a `TemplateResponse`, which is rendered here: https://github.com/nextcloud/server/blob/master/lib/private/AppFramework/Http/Dispatcher.php#L118-L119

During that call to `render()`, the "Cannot declare class GuzzleHttp\\Handler\\CurlFactory" fatal error happens. Here's a backtrace for that error:

    /var/www/html/lib/private/Template/Base.php - line 179:
	    include()
    /var/www/html/lib/private/Template/Base.php - line 151:
	    OC\Template\Base->load("/var/www/ht ... p", { remoteAddr ... "})
    /var/www/html/lib/private/legacy/template.php - line 207:
	    OC\Template\Base->fetchPage({ remoteAddr ... "})
    /var/www/html/lib/public/AppFramework/Http/TemplateResponse.php - line 157:
	    OC_Template->fetchPage({ remoteAddr ... "})
    /var/www/html/lib/private/AppFramework/Http/Dispatcher.php - line 119:
	    OCP\AppFramework\Http\TemplateResponse->render()
    /var/www/html/lib/private/AppFramework/App.php - line 118:
	    OC\AppFramework\Http\Dispatcher->dispatch(OCA\Richdocu ... {}, "index")
    /var/www/html/lib/private/AppFramework/Routing/RouteActionHandler.php - line 47:
	    OC\AppFramework\App::main("OCA\\Richdo ... r", "index", OC\AppFramew ... {}, { _route: "r ... "})
    (no filename shown?)
	    OC\AppFramework\Routing\RouteActionHandler->__invoke({ _route: "r ... "})
    /var/www/html/lib/private/Route/Router.php - line 297:
	    call_user_func(OC\AppFramew ... {}, { _route: "r ... "})
    /var/www/html/lib/base.php - line 989:
	    OC\Route\Router->match("/apps/richdocuments/index")
    /var/www/html/index.php - line 42:
	    OC::handleRequest()

**Comment by matthijskooijman** [comment]
The original `RequestException` is caught here: https://github.com/nextcloud/richdocuments/blob/9decb9f4edac15a558295306d5887a8eec8672a2/lib/Controller/DocumentController.php#L128-L142

There it translates to a `TemplateResponse`, which is rendered here: https://github.com/nextcloud/server/blob/master/lib/private/AppFramework/Http/Dispatcher.php#L118-L119

During that call to `render()`, the "Cannot declare class GuzzleHttp\\Handler\\CurlFactory" fatal error happens. Here's a backtrace for that error:

    /var/www/html/lib/private/Template/Base.php - line 179:
	    include()
    /var/www/html/lib/private/Template/Base.php - line 151:
	    OC\Template\Base->load("/var/www/ht ... p", { remoteAddr ... "})
    /var/www/html/lib/private/legacy/template.php - line 207:
	    OC\Template\Base->fetchPage({ remoteAddr ... "})
    /var/www/html/lib/public/AppFramework/Http/TemplateResponse.php - line 157:
	    OC_Template->fetchPage({ remoteAddr ... "})
    /var/www/html/lib/private/AppFramework/Http/Dispatcher.php - line 119:
	    OCP\AppFramework\Http\TemplateResponse->render()
    /var/www/html/lib/private/AppFramework/App.php - line 118:
	    OC\AppFramework\Http\Dispatcher->dispatch(OCA\Richdocu ... {}, "index")
    /var/www/html/lib/private/AppFramework/Routing/RouteActionHandler.php - line 47:
	    OC\AppFramework\App::main("OCA\\Richdo ... r", "index", OC\AppFramew ... {}, { _route: "r ... "})
    (no filename shown?)
	    OC\AppFramework\Routing\RouteActionHandler->__invoke({ _route: "r ... "})
    /var/www/html/lib/private/Route/Router.php - line 297:
	    call_user_func(OC\AppFramew ... {}, { _route: "r ... "})
    /var/www/html/lib/base.php - line 989:
	    OC\Route\Router->match("/apps/richdocuments/index")
    /var/www/html/index.php - line 42:
	    OC::handleRequest()


So it seems there's two problems:
 - When there is a certificate error, the error message template also fails to render. This is probably a problem in the richdocuments app.
 - When an error happens when rendering the error message, this is not caught and a fatal error happens. I believe this is a problem in the nextcloud core.

**Comment by danger89** [comment]
Well, actually I have the same issues, I'm using Nextcloud 14.0.1. I also get almost a empty page (it will not load) and get also a 500 error.

In my case the error pop-ups to me "cURL error 60: SSL certificate problem: unable to get local issuer certificate". However the Collabora Online url has a valid certificate from Let's Encrypt, which is:
https://office.melroy.org

It's a valid and trusted certificate. So I don't get it:
https://www.ssllabs.com/ssltest/analyze.html?d=office.melroy.org

![image](https://user-images.githubusercontent.com/628926/46618765-b15f3980-cb20-11e8-96d4-b0fbfa41b30d.png)


Nextcloud.log output:
```
{"reqId":"UKRJhhSw7FqmWgyoRyVE","level":3,"time":"2018-10-08T15:26:52+00:00","remoteAddr":"69.174.87.164","user":"melroy","app":"richdocuments","method":"GET",
"url":"\/apps\/richdocuments\/index?fileId=363551&requesttoken=m1K%2FXSrbAiDPdQ%2FkPPOm39536a2tH47eeg6TW7cumWY%3D%3A8xX6FWu5eBa2OSScbLeU7bAa3IKVW9q3T0CmaOZf1gE%3D","message":

{"Exception":"GuzzleHttp\\Exception\\RequestException","Message":"cURL error 60: SSL certificate problem: unable to get local issuer certificate (see http:\/\/curl.haxx.se\/libcurl\/c\/libcurl-errors.html)",

"Code":0,"Trace":[{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/CurlFactory.php","line":150,"function":"createRejection","class":"GuzzleHttp\\Handler\\CurlFactory","type":"::",
"args":[{"sink":{"__class__":"GuzzleHttp\\Psr7\\Stream"},"headers":[],"response":null,"request":{"__class__":"GuzzleHttp\\Psr7\\Request"},
"options":{"verify":"\/media\/data\/nextcloud\/files_external\/rootcerts.crt","synchronous":true,"handler":{"__class__":"GuzzleHttp\\HandlerStack"},
"allow_redirects":{"max":5,"protocols":["http","https"],"strict":false,"referer":false,"track_redirects":false},"http_errors":true,"decode_content":true,"cookies":false},
"errno":60,"onHeadersException":null,"__class__":"GuzzleHttp\\Handler\\EasyHandle"},{"errno":60,"error":"SSL certificate problem: unable to get local issuer certificate","url":"https:\/\/office.melroy.org\/hosting
\/discovery","content_type":null,"http_code":0,"header_size":0,"request_size":0,"filetime":-1,"ssl_verify_result":20,"redirect_count":0,"total_time":0.14769699999999997,"namelookup_time":0.12499800000000005,"connect_time":0.12587099999999996,"pretransfer_time":0,"size_upload":0,"size_download":0,"speed_download":0,"speed_upload":0,"download_content_length":-1,"upload_content_length":-1,"starttransfer_time":0,"redirect_time":0,"redirect_url":"","primary_ip":"85.145.170.136","certinfo":[],"primary_port":443,"local_ip":"192.168.1.20","local_port":59672}]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/CurlFactory.php","line":103,"function":"finishError","class":"GuzzleHttp\\Handler\\CurlFactory","type":"::","args":[{"__class__":"GuzzleHttp\\Handler\\CurlHandler"},{"sink":{"__class__":"GuzzleHttp\\Psr7\\Stream"},"headers":[],"response":null,"request":{"__class__":"GuzzleHttp\\Psr7\\Request"},"options":{
"verify":"\/media\/data\/nextcloud\/files_external\/rootcerts.crt",
"synchronous":true,"handler":{"__class__":"GuzzleHttp\\HandlerStack"},"allow_redirects":{"max":5,"protocols":["http","https"],"strict":false,"referer":false,"track_redirects":false},"http_errors":true,"decode_content":true,"cookies":false},"errno":60,"onHeadersException":null,"__class__":"GuzzleHttp\\Handler\\EasyHandle"},{"__class__":"GuzzleHttp\\Handler\\CurlFactory"}]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/CurlHandler.php","line":43,"function":"finish","class":"GuzzleHttp\\Handler\\CurlFactory","type":"::","args":[{"__class__":"GuzzleHttp\\Handler\\CurlHand
ler"},{"sink":{"__class__":"GuzzleHttp\\Psr7\\Stream"},"headers":[],"response":null,"request":{"__class__":"GuzzleHttp\\Psr7\\Request"},"options":{"verify":"\/media\/data\/nextcloud\/files_external\/rootcerts.crt","synchronous":true,"handler":{"__class__":"GuzzleHttp\\HandlerStack"},"allow_redirects":{"max":5,"protocols":["http","https"],"strict":false,"referer":false,"track_redirects":false}
,"http_errors":true,"decode_content":true,"cookies":false},"errno":60,"onHeadersException":null,"__class__":"GuzzleHttp\\Handler\\EasyHandle"},
{"__class__":"GuzzleHttp\\Handler\\CurlFactory"}]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/Proxy.php","line":28,"function":"__invoke","class":"GuzzleHttp\\Handler\\CurlHandler","type":"->","args":["*** sensitive parameter replaced *
**","*** sensitive parameter replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/Proxy.php","line":51,"function":"GuzzleHttp\\Handler\\{closure}","class":"GuzzleHttp\\Handler\\Proxy","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/PrepareBodyMiddleware.php","line":37,"function":"GuzzleHttp\\Handler\\{closure}","class":"GuzzleHttp\\Handler\\Proxy","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Middleware.php","line":30,"function":"__invoke","class":"GuzzleHttp\\PrepareBodyMiddleware","type":"->","args":["*** sensitive parameter replaced ***","*** sensitive parameter replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/RedirectMiddleware.php","line":70,"function":"GuzzleHttp\\{closure}","class":"GuzzleHttp\\Middleware","type":"::","args":["*** sensit
ive parameters replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Middleware.php","line":59,"function":"__invoke","class":"GuzzleHttp\\RedirectMiddleware","type":"->","args":["*** sensitive parameter replaced ***","*** sensitive parameter replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/HandlerStack.php","line":67,"function":"GuzzleHttp\\{closure}","class":"GuzzleHttp\\Middleware","type":"::","args":["*** sensitive parameters replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Client.php","line":277,"function":"__invoke","class":"GuzzleHttp\\HandlerStack","type":"->","args":["*** sensitive parameter replaced ***","*** sensitive parameter replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Client.php","line":125,"function":"transfer","class":"GuzzleHttp\\Client","type":"->","args":["*** sensitive parameter replaced ***","*** sensitive parameter replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Client.php","line":131,"function":"requestAsync","class":"GuzzleHttp\\Client","type":"->","args":["get",{"__class__":"GuzzleHttp\\Psr7\\Uri"},{"verify":"\/media\/data\/nextcloud\/files_external\/rootcerts.crt","synchronous":true,"handler":{"__class__":"GuzzleHttp\\HandlerStack"},"allow_redirects":{"max":5,"protocols":["http","https"],"strict":false,"referer":false,"track_redirects":false},"http_errors":true,"decode_content":true,"cookies":false,"_conditional":{"User-Agent":"GuzzleHttp\/6.2.1 curl\/7.52.1 PHP\/7.1.22-1+0~20181001133718.6+stretch~1.gbp113f3b"}}]},{"file":"\/var\/www\/nextcloud\/html\/lib\/private\/Http\/Client\/Client.php","line":161,"function":"request","class":"GuzzleHttp\\Client","type":"->","args":["get","https:\/\/office.melroy.org\/hosting\/discovery",
{"verify":"\/media\/data\/nextcloud\/files_external\/rootcerts.crt","synchronous":true}]},{"file":"\/var\/www\/nextcloud\/html\/apps\/richdocuments\/lib\/WOPI\/DiscoveryManager.php","line":84,"function":"get","class":"OC\\Http\\Client\\Client","type":"->","args":["https:\/\/office.melroy.org\/hosting\/discovery"]},{"file":"\/var\/www\/nextcloud\/html\/apps\/richdocuments\/lib\/WOPI\/Parser.php","line":41,"function":"get","class":"OCA\\Richdocuments\\WOPI\\DiscoveryManager","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/html\/apps\/richdocuments\/lib\/TokenManager.php","line":148,"function":"getUrlSrc","class":"OCA\\Richdocuments\\WOPI\\Parser","type":"->","args":["application\/vnd.oasis.opendocument.text"]},{"file":"\/var\/www\/nextcloud\/html\/apps\/richdocuments\/lib\/Controller\/DocumentController.php","line":179,"function":"getToken","class":"OCA\\Richdocuments\\TokenManager","type":"->","args":["*** sensitive parameters replaced ***"]}
,{"file":"\/var\/www\/nextcloud\/html\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":166,"function":"index","class":"OCA\\Richdocuments\\Controller\\DocumentController","type":"->","args":["*** sensitive parameter replaced ***"]},{"file":"\/var\/www\/nextcloud\/html\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Richdocuments\\Controller\\DocumentController"},"index"]},{"file":"\/var\/www\/nextcloud\/html\/lib\/private\/AppFramework\/App.php","line":118,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Richdocuments\\Controller\\DocumentController"},"index"]},{"file":"\/var\/www\/nextcloud\/html\/lib\/private
\/AppFramework\/Routing\/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Richdocuments\\Controller\\DocumentController","index",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"_route":"richdocuments.document.index"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"_route":"richdocuments.document.index"}]},{"file":"\/var\/www\/nextcloud\/html\/lib\/private\/Route\/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"_route":"richdocuments.document.index"}]},{"file":"\/var\/www\/nextcloud\/html\/lib\/base.php","line":987,"function":"match","class":"OC\\Route\\Router","type":"->","args":["\/apps\/rich
documents\/index"]},{"file":"\/var\/www\/nextcloud\/html\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"\/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/CurlFactory.php","Line":187,"CustomMessage":"--"},"userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64; rv:63.0) Gecko\/20100101 Firefox\/63.0","version":"14.0.1.1"}                
{"reqId":"UKRJhhSw7FqmWgyoRyVE","level":3,"time":"2018-10-08T15:26:52+00:00","remoteAddr":"69.174.87.164","user":"melroy","app":"PHP","method":"GET","url":"\/apps\/richdocuments\/index?fileId=363551&requesttoken=m1K%2FXSrbAiDPdQ%2FkPPOm39536a2tH47eeg6TW7cumWY%3D%3A8xX6FWu5eBa2OSScbLeU7bAa3IKVW9q3T0CmaOZf1gE%3D","message":"Cannot declare class GuzzleHttp\\Handler\\CurlFactory, because the name is already in use at \/var\/www\/nextcloud\/html\/3rdparty\/guzzlehttp\/guzzle\/src\/Handler\/CurlFactory.php#16","userAgent":"Mozilla\/5.0 (Windows NT 6.1; Win64; x64; rv:63.0) Gecko\/20100101 Firefox\/63.0","version":"14.0.1.1"}                                                                                                                                                                                          
```

**Comment by SeriousM** [comment]
I have the same exception but a bit different setup: my nextcloud v14 is behind a reverse proxy and does not have access to the certificates at all.
Is this information helpful?

**Comment by danger89** [comment]
Mine is also behind a reverse proxy (nginx web server). How do you mean it has no access to the certificates?

**Comment by xundeenergie** [comment]
I have the same problem.
Collabora-Code is not useable with nextcloud 14.0.3

**Comment by lightweight** [comment]
This problem seems to affect me, too. Error I get is: 
```
[error] 6#6: *5826 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot declare class GuzzleHttp\Handler\CurlFactory, because the name is already in use in /var/www/html/3rdparty/guzzlehttp/guzzle/src/Handler/CurlFactory.php on line 16" while reading response header from upstream, client: 172.25.0.1, server: , request: "GET /apps/richdocuments/index?fileId=139048&requesttoken=RH%2FPowxvvpK6DjzZVvrXo%2BCmP0EopcYemvQpnxeOcnc%3D%3AE1C6wkQtjvn0Qg7rGI2N867sDzhwzZ5QrrsRxUHaS0I%3D HTTP/1.0", upstream: "fastcgi://172.25.0.3:9000", host: "cloud.lane.net.nz"
```
I'm running 14.0.3 via Docker for both NextCloud and Collabora, using this setup: https://tech.oeru.org/installing-nextcloud-and-collabora-office-online-docker-ubuntu-1604

**Comment by plepps** [comment]
I had the same issue - it was because my NC was not a native subdomain. "https://server.foo.bar/nc"
I changed NC to run on this URL "https://nc.foo.bar" and start a new collabbora docker with this new domain and it works perfectly now!

**Comment by stefan-kaestle** [comment]
In contrast to the previous comment: I got it working with nextcloud running in a subfolder "http://server.foo.bar/nc" just fine in NC 15 with Collabora Online 4.0.0.

**Comment by danger89** [comment]
Oowh, well if we just auto close everything, this solves nothing.

**Comment by ggeorgg** [comment]
Thanks @kesselb ! I just added the "verify false" option and I can now open files in collabora in nextcloud! 
I have been struggling for a while because I wanted to test it first in my virtualbox with self signed certificates...

**Comment by kevdogg** [comment]
What's the root cause of this problem?  I'm running the nextcloud instance at nextcloud.<domain>.com and my main website at <domain>.com with Apache24. When using the Let'sEncrypt Certificates I can access the main website just fine and even the nextcloud.<domain>.com site without any problems.  If the ssl certs were not valid wouldn't this cause an issue with these sites?  Why does the nextcloud/collabora connection then find the certs to be invalid?  I don't understand.

**Comment by kesselb** [comment]
If you access nextcloud or collabora your client accept the certificates (lets encrypt is trusted by your browser or os). Nextcloud and collabora use https for internal connections (this traffic is not going through your client) and this connection fails because of some ssl issues.

You could verify this like:

1. connect to your nextcloud instance by ssh
2. try `curl https://mycollaborainstance`

Step 2 should fail with some ssl related warnings because the connection appears to be insecure. You may fix this by adding https://letsencrypt.org/certificates/ (the root certificate) to your local certificate store.

**Comment by kevdogg** [comment]
Hmmmm -- not what I expected
My cert has one domain -- <domain>.com and two subdomain nextcloud.<domain>.com and office.<domain>.com. The two subdomain are on different machines

I ssh into nextcloud instance and then ran curl
curl https://office.<domain>.com
curl: (7) Failed to connect to office.<domain>.com port 443: Connection refused

This is unexpected since I setup a simple index.html file on office.<domain>.com and can have it display in the browswer

However if I do the following:
curl https://office.gohilton.com:9980
OK

I also disabled the news reader and it didn't seem to have any effect.  I can disable more apps if it helps.

**Comment by kesselb** [comment]
> When an error happens when rendering the error message, this is not caught and a fatal error happens. I believe this is a problem in the nextcloud core.

@matthijskooijman fix https://github.com/nextcloud/richdocuments/pull/465 (more a workaround than a fix)
